scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/types/ConverterProvider.scala

```diff
@@ -171,13 +171,14 @@ private[types] object ConverterProvider {
       val provider: OverrideTypeProvider =
         OverrideTypeProviderFinder.getProvider
       tpe match {
-        case t if provider.shouldOverrideType(c)(t) => q"$tree.toString"
-        case t if t =:= typeOf[Boolean]             => tree
-        case t if t =:= typeOf[Int]                 => q"$tree.toLong"
-        case t if t =:= typeOf[Long]                => tree
-        case t if t =:= typeOf[Float]               => q"$tree.toDouble"
-        case t if t =:= typeOf[Double]              => tree
-        case t if t =:= typeOf[String]              => tree
+        case t if provider.shouldOverrideType(c)(t)                     => q"$tree.toString"
+        case t if t =:= typeOf[Boolean]                                 => tree
+        case t if t =:= typeOf[Int]                                     => q"$tree.toLong"
+        case t if t =:= typeOf[Long]                                    => tree
+        case t if t =:= typeOf[Float]                                   => q"$tree.toDouble"
+        case t if t =:= typeOf[Double]                                  => tree
+        case t if t =:= typeOf[String]                                  => tree
+        case t if t =:= typeOf[com.fasterxml.jackson.databind.JsonNode] => tree
```

Comment:
I don't think we should accept `JsonNode` here

Reply:
I don't actually need it, but I supposed one may want to provide an already-valid JSON. I will remove it.